docs/topics/migrations.txt

```diff
@@ -629,27 +632,67 @@ to be able to write your own, see the :doc:`migration operations reference
 </ref/migration-operations>` and the "how-to" on :doc:`writing migrations
 </howto/writing-migrations>`.
 
-.. _migration-squashing:
 
-Squashing migrations
-====================
+Optimizing migrations
+=====================
 
-You are encouraged to make migrations freely and not worry about how many you
-have; the migration code is optimized to deal with hundreds at a time without
-much slowdown. However, eventually you will want to move back from having
-several hundred migrations to just a few, and that's where squashing comes in.
+Django developers are free to create as migrations as required and do not have
+to worry about the number of migrations created as Django is optimized to deal
+with hundreds at a time without much slowdown.  However, there may come a time when migrations will become so numerous that it
+will start to have a noticeable impact on various tasks like running tests on
+CI.
 
-Squashing is the act of reducing an existing set of many migrations down to
-one (or sometimes a few) migrations which still represent the same changes.
+There are a few steps that developers can take to mitigate or remediate this by
+reducing the amount of work that migrations has to do:
+
+
```

Comment:
There are many unnecessary blank lines in this draft, please remove them.
```suggestion

```

Reply:
ay caramba nobody's reading my "please ignore finer details" 😅

i didn't want to waste people's time with this, i was mostly looking for feedback on whether the reorganisation was a good idea. once we've confirmed this is the approach we want then i was going to go back and clean up all this 😜